pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -169,9 +169,15 @@ func (c *Cluster) Start() error {
 	// If the mgr is newly created, wait for it to start before continuing with the service and
 	// module configuration
 	for _, d := range deploymentsToWaitFor {
-		if err := waitForDeploymentToStart(c.clusterInfo.Context, c.context, d); err != nil {
+		mgrLabel := fmt.Sprintf("%s=%s", controller.DaemonIDLabel, cephv1.KeyMgr)
```

Comment:
This will query for the label `ceph_daemon_id=mgr`, but that doesn't return any pods. Looks like what we need is `ceph_daemon_type=mgr`.

Reply:
Fixed